lib/scope.js

```diff
@@ -2503,6 +2514,29 @@ module.exports = {
       await scope.detectDownloadComplete( scope );
     },  // Ends scope.steps.download()
 
+    compare_pdfs: async function (scope, {existing_pdf_path, new_pdf_path}) {
+      let existing_paths = await scope.findFiles(scope, {to_find_names: [existing_pdf_path]});
+      if (existing_paths.length == 0) {
+        // The row wasn't used
+        let msg = `Could not find the existing PDF at ${ existing_pdf_path }`;
+        expect(existing_paths.length, msg).to.be.above(0);
```

Comment:
Sorry, forgot about adding to the report. IMO, yeah, a warning in `findFiles`, and but potentially error in the function that calls `findFiles` is a good option.

TBH doing what we do with the accessibility stuff is a good idea, and not hard to implement. It will add to #748, but whatever solution we have for that would also fix this as well. I'll try it out, if it's taking too long, I'll make an issue for it.

Reply:
Found a good solution for this, I think it works out really well, except for the above mentioned extra screenshot with #748, but this and the accessibility code is very closely related, and the solution to one should fix the other.